src/index.html

```diff
@@ -6,13 +6,65 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <title>Document</title>
```

Comment:
The `title` element should have a more descriptive title relevant to the content of the page.